addons/stock/wizard/stock_inventory_warning.xml

```diff
@@ -33,4 +33,21 @@
             </form>
         </field>
     </record>
+
+    <record id="inventory_warning_set_to_zero_view" model="ir.ui.view">
+        <field name="name">inventory.set_to_zero.warning.view</field>
+        <field name="model">stock.inventory.warning</field>
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <form>
+                <div>
+                    Some selected lines already have quantities set, they will be ignored.
+                </div>
+                <footer>
+                    <button name="action_set_to_zero" string="Continue" type="object" class="btn-primary" data-hotkey="q"/>
+                    <button name="cancel_button" string="Discard" class="btn-secondary" special="cancel" data-hotkey="z"/>
+                </footer>
+            </form>
+        </field>
+    </record>
```

Comment:
Instead of the wizard, it would be great to just display a notification. It will avoid extra rpc, ...

Reply:
@amoyaux 

Are you saying the functionality of setting the value to 0 directly on the button click and notifying the user?

Reply:
Just remove that part and don't notify the user